modules/microshift-cni.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-cni_{context}"]
+= About the OVN-Kubernetes CNI network provider
+
+//Q: this module is not done; included for the sake of the build
+//[IMPORTANT]
+//====
+//The default CNI configuration is intended to match the developer environment described in MicroShift Development Environment on RHEL 8.
+//====
+//Q: What's the default production configuration?
+
+{product-title} uses OVN-Kubernetes, a virtualized network for pods and services. The OVN-Kubernetes Container Network Interface (CNI) plug-in is the network provider for the cluster. OVN-Kubernetes is based on Open Virtual Network (OVN) and provides an overlay-based networking implementation.
+
+A cluster that uses the OVN-Kubernetes network provider also runs Open vSwitch (OVS) on the node. OVN configures OVS on the node to implement the declared network configuration. OVS runs as a systemd service on the {product-title} node. The OVS RPM package is installed as a dependency to the `microshift-networking` RPM package. OVS is started immediately when the `microshift-networking` RPM is installed.
+
+By default, three performance optimizations are applied to OVS services to minimize resource consumption:
+
+* CPU affinity to `ovs-vswitchd.service` and `ovsdb-server.service`
+* `no-mlockall` to `openvswitch.service`
+* Limit handler and `revalidator` threads to `ovs-vswitchd.service`
+
+With {product-title}, each node is contained to a single machine and single {product-title}, so each deployment will require individual configuration, pods, and settings.
```

Comment:
I'm not sure we need to re-emphasize this.

@zshi-redhat or @mangelajo what do you think? Can we remove this sentence from user docs?

Reply:
Agree to remove, this single-node principle applies to all microshift components.

Reply:
@dhellmann @zshi-redhat Hmm, we don't say this so explicitly anywhere else in the docs. I think it's useful somewhere. Perhaps integrated into the Understanding section somehow?